src/solid-shapes/include/iDynTree/InertialParametersSolidShapesHelpers.h

```diff
@@ -32,8 +33,25 @@ namespace iDynTree
 bool estimateInertialParametersFromLinkBoundingBoxesAndTotalMass(const double totalMass,
                                                                  iDynTree::Model& model,
                                                                  VectorDynSize& estimatedInertialParams);
+/**
+ * @brief Compute bounding box from a solid shape object
+ *
+ * @param[in] geom The link collision as a iDynTree solid shape object
+ * @param[out] box The bounding box for the solid shape object
+ * @return true if all went well, false otherwise .
+ *
+ * @note If the shape is an ExternalMesh type, this function requires IDYNTREE_USES_ASSIMP to be true, otherwise it always returns false.
+ * @note If the shape is an ExternalMesh type, AABB algorithm is used to compute the bounding box form the mesh vertices.
+ */
+bool BBFromShape(SolidShape* geom, Box& box);
```

Comment:
To clarify, the AABB is not a specific algorithm to compute a bounding box, but it means *axis aligned* bounding box, meaning that it may not be the minimal value bounding box. I would try to clarify this in the method description, specifying that the returned BoundingBox is neither unique nor the minimal volume one for externalmeshes. 

For coherence with the rest of the code and the convention used for free functions, I would call it `computeBoundingBoxFromShape`. 

I would also pass the first argument as `const SolidShape &` instead of using a raw pointer.

Reply:
I would also probably clarify if the frame to which the axis are aligned for AABB are the frame of the shape frame or the one of the link frame.

Reply:
Done in https://github.com/robotology/idyntree/pull/801/commits/299c67d8d7ac0d6b11148ea6d6d35b2c959e8711